### DISCUSSION TITLE
Socket.io is not connecting when using /socket.io/?EIO=4&transport=websocket

### DISCUSSION THREAD

**Discussion by mahasheikhignite** [socketio/socket.io#4671] [Q&A] [answered]
I am using nodejs backend and nodejs ejs frontend and my deployment server is apache. I have successfully run flow on localhost but when I move the code to production it gives me  400 bad request error on https://serverURL/socket.io/?EIO=4&transport=websocket but it send response on https://serverURL/socket.io/?EIO=4&transport=polling 
0{"sid":"bpyERQ_HzRN51GyiAAAA","upgrades":["websocket"],"pingInterval":6000,"pingTimeout":3000,"maxPayload":1000000}
I also tries to add cors on server side but failed.

I am stuck can any one please help. Thanks in advance.

**Comment by darrachequesne** [comment, marked as answer]
Hi! Please check the documentation for apache: https://socket.io/docs/v4/reverse-proxy/#apache-httpd
